packages/circuits/scripts/prepare.js

```diff
@@ -31,6 +31,12 @@ try {
     )
 } catch (_) {}
 
+// remove test keys
+const zkPath = path.join(__dirname, '../dist/zksnarkBuild')
+fs.readdirSync(zkPath)
+    .filter((filename) => /^bigComparators/.test(filename))
```

Comment:
it would be better if we set the test circuits in a list if we have many test circuits in the future.
but it would be fine to be in this PR.